### PR TITLE
[SYCLomatic] Add option "-DMKL_ILP64" to CMAKE_CXX_FLAGS in cmake help function

### PR DESCRIPTION
**Pull request by tomflinda**
Signed-off-by: chenwei.sun <chenwei.sun@intel.com>

### CODE REVIEW

**Review by zhiweij1 — APPROVED** [review]

File: clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   In: project(${arg0})
   Out: |
    project(${arg0})
-   set(CMAKE_CXX_FLAGS "\${CMAKE_CXX_FLAGS} -fsycl")
+   set(CMAKE_CXX_FLAGS "\${CMAKE_CXX_FLAGS} -fsycl -DMKL_ILP64")
```
Comment (zhimingwang36) [inline clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml:18]: what is the hint to add -DMKL_ILP64?

Comment (zhiweij1) [inline clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml:18]:
```suggestion
   set(CMAKE_CXX_FLAGS "\${CMAKE_CXX_FLAGS} -fsycl -DMKL_ILP64 -qmkl")
```

Comment (wenhuiNi) [reply clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml:18]: @zhiweij1 Are -qmkl and  -lmkl_sequential -lmkl_core -lmkl_sycl the same thing?
Do we need change -DMKL_ILP64 to -lmkl_intel_lp64?

Comment (tomflinda) [reply clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml:18]: Will add this option when needed it.

**Comment by tomflinda** [comment]
Close this PR.

File: clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml (hunk 1 above)
Comment (zhiweij1) [reply clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml:18]: I think `-qmkl=sequential` has same effect as `-lmkl_sycl -lmkl_intel_ilp64 -lmkl_sequential -lmkl_core`

Comment (zhiweij1) [reply clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml:18]: If we use `-lmkl_intel_ilp64`, we should add `-DMKL_ILP64`.
If we use `-lmkl_intel_lp64`, we should add `-DMKL_LP64`.

**Pull request closed without merging**